jupyterlab-dataops/resources/cloudbeaver/set_metadata.py

```diff
@@ -1,6 +1,10 @@
 from google.cloud import compute_v1
```

Comment:
when this script is executed?

Reply:
it's called from Bash script at startup of the image. Please check CMD in Dockerfile.